spec/lib/api/2.0/tasks-spec.js

```diff
@@ -37,8 +37,26 @@ describe('Http.Api.Tasks', function () {
           });
     });
 
+    afterEach('restoring stubs', function() {
+        function resetStubs(obj) {
+            _(obj).methods().forEach(function (method) {
+                if (obj[method] && obj[method].reset) {
+                    obj[method].reset();
+                }
+            }).value();
+        }
+
+        resetStubs(taskProtocol.activeTaskExists);
+        resetStubs(taskProtocol.requestCommands);
+        resetStubs(taskProtocol.respondCommands);
+        resetStubs(tasksApiService.getNode);
+        resetStubs(lookupService.ipAddressToMacAddress);
+    });
+
     after('stop HTTP server', function () {
-        return helper.stopServer();
+        return helper.reset().then(function(){
```

Comment:
@yyscamper I added this because I saw the database was being cleared and then injecting Views so I just wanted to make sure that no stale Views were being carried over from this test.  Some tests don't use Views so that's probably why you don't see this in all of the tests